srcs/layouts/latn_qwerty_da.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Danish)" script="latin">
+  <row>
+    <key key0="q" key4="esc" key2="1"/>
+    <key key0="w" key2="2" key3="`" key4="\@"/>
+    <key key0="e" key2="3" key4="\#" key3="€"/>
+    <key key0="r" key2="4" key4="$"/>
+    <key key0="t" key2="5" key3="&amp;" key4="|"/>
+    <key key0="y" key2="6" key3="^" key4="%"/>
+    <key key0="u" key2="7" key3="~" key4="\\"/>
+    <key key0="i" key2="8" key3="*" key4="/"/>
+    <key key0="o" key2="9" key3="+" key4="-"/>
+    <key key0="p" key2="0" key3="=" key4="f11_placeholder"/>
+    <key key0="å" key1="\?" key3="!" key4="f12_placeholder"/>
+  </row>
+  <row>
+    <key key0="a" key2="tab"/>
+    <key key0="s" key1="accent_ring" key2="loc §" key3="loc ß"/>
+    <key key0="d" key3="accent_aigu"/>
+    <key key0="f" key1="accent_trema"/>
+    <key key0="g"/>
+    <key key0="h"/>
+    <key key0="j" key1="{" key4="}"/>
+    <key key0="k" key1="[" key4="]"/>
+    <key key0="l" key1="(" key4=")"/>
+    <key key0="æ" key1="&lt;" key4="&gt;"/>
+    <key key0="ø" key1="'" key3="&quot;"/>
+  </row>
+  <row>
+    <key width="1.5" key0="shift" key2="loc capslock"/>
+    <key key0="z"/>
+    <key key0="x" key2="loc †"/>
+    <key key0="c"/>
+    <key key0="v"/>
+    <key key0="b" key2=";" key4=","/>
+    <key key0="n" key2=":" key4="."/>
+    <key key0="m" key2="-" key4="_"/>
+    <key width="1.0" key0="." key1="!" key2="\?" key3="'" key4=","/>
```

Comment:
Some keys are present twice on the layout: `!, ', ,, -, ., ?`.

You can check some properties of the layout with `gradle test`. This will update the file `check_layout.output`, that should be committed.

Reply:
I will have a look at this. Thanks!

Reply:
Should be resolved now!